ca/ca_test.go

```diff
@@ -113,6 +113,12 @@ var (
 
 	// OIDExtensionCTPoison is defined in RFC 6962 s3.1.
 	OIDExtensionCTPoison = asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 11129, 2, 4, 3}
+
+	issuanceModes = []IssuanceMode{
```

Comment:
Could you use the `IssuanceMode` field names in the instantiations in `issuanceModes`? I think it will make the two `bool` values easier to understand without having to cross-reference with the later `IssuanceMode` struct definition.